src/fastcs/mapping.py

```diff
@@ -46,15 +46,17 @@ def _get_single_mapping(controller: BaseController) -> SingleMapping:
         attr = getattr(controller, attr_name)
         match attr:
             case WrappedMethod(fastcs_method=fastcs_method):
-                match fastcs_method:
-                    case Put():
-                        put_methods[attr_name] = fastcs_method
-                    case Scan():
-                        scan_methods[attr_name] = fastcs_method
-                    case Command():
-                        command_methods[attr_name] = fastcs_method
+                if fastcs_method.enabled:
+                    match fastcs_method:
+                        case Put():
+                            put_methods[attr_name] = fastcs_method
+                        case Scan():
+                            scan_methods[attr_name] = fastcs_method
+                        case Command():
+                            command_methods[attr_name] = fastcs_method
             case Attribute():
-                attributes[attr_name] = attr
+                if attr.enabled:
+                    attributes[attr_name] = attr
```

Comment:
This is getting quite indented now. It could possibly be simplified with something like:
```suggestion
        match attr:
            case WrappedMethod(fastcs_method=Put(enabled=True) as put_method):
                put_methods[attr_name] = put_method
            case WrappedMethod(fastcs_method=Scan(enabled=True) as scan_method):
                    scan_methods[attr_name] = scan_method
            case WrappedMethod(fastcs_method=Command(enabled=True) as command_method):
                    command_methods[attr_name] = command_method
            case Attribute(enabled=True):
                attributes[attr_name] = attr
```